Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Luciferbug.xml

```diff
@@ -20,14 +20,14 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_LuciferBug"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>0.5</ArmorRating_Blunt>
+					<ArmorRating_Blunt>12</ArmorRating_Blunt>

```

Comment:
Leave it. They're intentionally fragile creatures that are dangerous if damaged.